rosidl_generator_cpp/resource/msg.hpp.template

```diff
@@ -15,6 +15,7 @@
 #ifndef __@(spec.base_type.pkg_name)__@(subfolder)__@(get_header_filename_from_msg_name(spec.base_type.type))__hpp__
 #define __@(spec.base_type.pkg_name)__@(subfolder)__@(get_header_filename_from_msg_name(spec.base_type.type))__hpp__
 
+#include "@(spec.base_type.pkg_name)/@(subfolder)/@(get_header_filename_from_msg_name(spec.base_type.type))__meta.hpp"
```

Comment:
I'd say that is should, since in the use case where the classes in rclcpp, like subscription, are using the traits, the user would then need to know when that's going to happen and preemptively include the traits for the right message as the rclcpp classes cannot include the file when it is needed. So including it in the default message header seems like the best strategy in my opinion.